### PR TITLE
Step 1: Custom DOM manipulation challenge

### PR DESCRIPTION
**Pull request by fatihtutar**
ex 1 part 1

### CODE REVIEW

**Review by bitsagarob — CHANGES_REQUESTED** [review]
very good, now also make the todo list bonus exercise from this file

**Review by bitsagarob — APPROVED** [review]
Great job! Now onto the original homework, this will be more difficult. But you'll be able to!

File: Week1/homework/app.js [hunk 1]
```diff
@@ -1,11 +1,123 @@
...
+  
+ // 1.3 Remove the temporary console.log from step 1.1. Make a function (or functions) that generate a ul with li elements for each book ID in the array using a for loop. 
+ 
+  const listContainer= document.getElementById('books');
+
+  // Make the list
+  const listElement = document.createElement('ul');
+
+  // Add it to the page
+  listContainer.appendChild(listElement);
```
Comment (segersrobbertKlarrio) [inline Week1/homework/app.js:40]: if it appears the `listContainer` variable has no value. Verify by console logging that variable and fix the probable issue at line 34 where you fetch the element

File: Week1/homework/index.html [hunk 2]
```diff
@@ -1 +1,14 @@
...
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta charset="UTF-8" />
+    <script src="app.js"></script>
+    <title>Book Blog</title>
+  </head>
+  <body>
+    <div id= "books"></div>
```
Comment (segersrobbertKlarrio) [inline Week1/homework/index.html]: there is a typo here

File: Week1/homework/app.js (hunk 1 above)
Comment (fatihtutar) [reply Week1/homework/app.js:40]: It seems that the problem is solved. Thanks

File: Week1/homework/app.js [hunk 3]
```diff
@@ -1,11 +1,123 @@
...
+
+  // Set up a loop that goes through the items in listItems one at a time
+  const numberOfListItems = bookTitles.length;
+  function bookList() {
+  for (let i = 0; i < numberOfListItems; ++i) {
```
Comment (bitsagarob) [inline Week1/homework/app.js:45]: indentation of for loop

**Review by bitsagarob — APPROVED** [review]
Great job, beautiful code & commenting. Try to finish the books exercise!

**Pull request closed without merging**